lib/launcher.js

```diff
@@ -58,8 +58,8 @@ class AppiumLauncher {
         }
     }
 
-    _startAppium (): Promise<child_process$ChildProcess> {
-        return new Promise((resolve, reject) => {
+    _startAppium (): Promise<child_process$ChildProcess> { // eslint-disable-line camelcase
+        return new Promise((resolve: any, reject: any) => {
```

Comment:
`resolve` and `reject` should not be typed as `any`. Their types are function which takes one argument and returns nothing.

Reply:
Fixed